molecule/default/verify.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Check whether packages are installed
+  hosts: all
+  become: yes
+
+  tasks:
+
+    - name: Verify utilities are available
+      command: >
+        {{ item }}
+        with_items:
+      with_items:
+        - git --version
+        - curl --version
+        - htop --version
+        - iotop --version
+        - make --version
+        - multitail -h
```

Comment:
The `-h` flag shows help, but also fails with error. Use `-V` instead.

Reply:
I had tried -V, it did't help.

Reply:
Don’t worry it will. We just need to get those env vars into the right place: we can’t use them inline like we would in a shell. Check out https://docs.ansible.com/ansible/latest/user_guide/playbooks_environment.html#setting-the-remote-environment-in-a-task